### ISSUE TITLE
Add CryptoServer GRPC

### ISSUE THREAD

**Issue by bcncpp** [zenforcode/keyfort#10]
As a Developer
I want a software cryptography server so that we are able to encipher data and store in a key-value store.

    User Persona: [Provide more details about the user, including their role, goals, and how they interact with the system. For example, “A returning customer who primarily shops on mobile devices.”]
    Context or Problem Statement: [Describe the user’s pain point or the situation leading to this request. Highlight why it’s important and the problem being solved.]

### Acceptance Criteria

Detailed list of conditions or requirements that the implementation must fulfill. Each criterion must be clear, measurable, and testable.

    [Condition 1]:  Cyptoserver can encipher the keys
    [Condition 2]: 
    [Condition 3]: [Example: “The user interface must adapt to different screen sizes.”]

### Workflow or Steps for Implementation

Provide a step-by-step breakdown of how the user story should be implemented:

    [Step 1: What needs to be done first? For example, “Design the email template for the password reset email.”]
    [Step 2: Next steps, e.g., “Integrate the front-end form with the backend API for password reset requests.”]
    [Step 3: Testing scenarios or edge cases to consider, e.g., “Ensure invalid email addresses return an appropriate error message.”]

### Technical Requirements

    [Specific technical needs or constraints, such as APIs to be integrated, frameworks to use, or performance standards.]
    [Example: “The password reset email should use an existing SMTP service and comply with GDPR requirements for user data privacy.”]

### Definition of Done (DoD)

Provide a detailed checklist of what must be achieved for the story to be considered complete.

    [Code is implemented, reviewed, and merged into the main branch.]
    [Unit tests and integration tests are written and pass successfully.]
    [Functional requirements match acceptance criteria.]
    [No high-priority bugs in testing or staging.]
    [Deployed to staging or production, as agreed.]

### Story Points and Effort Estimation

    Story Points: [Estimated effort or complexity of the task, e.g., 5 points.]
    Expected Duration: [Rough time estimate for completion, e.g., 3–5 days.]

### Priority and Impact

    Priority Level: [Low, Medium, High, Critical.]
    Impact Analysis: [Details on how this story will affect the product, user experience, or business goals.]

### Dependencies and Risks

    Dependencies: [List any related tasks, systems, or teams that must complete work before this story can proceed.]
    Risks: [Identify potential challenges or blockers, such as tight deadlines, technical uncertainties, or resource limitations.]

### Additional Notes or Context

    [Include any supporting information, such as mockups, diagrams, or links to related documentation.]
    [Questions or open items for discussion with the team.]
